pg_backup_api/pg_backup_api/run.py

```diff
@@ -94,32 +158,55 @@ def recovery_operation(args):
     end_time = server_ops.time_event_now()
     content["success"] = success
     content["end_time"] = end_time
-    content["output"] = output.decode() if isinstance(output, bytes) else output
+    content["output"] = output.decode() if isinstance(output, bytes) \
```

Comment:
I don't see any advantages to splitting the line here. It's not like the original line is terribly long that would make sense to split it.

Reply:
I only did that to silence `flake8`.
I'm not sure if we intend to add a linter to the project later to check the code on pushed commits?
I can revert that change and silence the flake8 warning if you prefer.

Reply:
We could add black/flake8 to our workflow if you like. I usually try to prioritise readability than form. The above splitting I think proves my point.

Reply:
We can convert that into 2 lines or write a method to sanitise content{"output"] so we can silence flake8 and will be easy to read IMO.

Reply:
Sounds good. I'm reverting the change for now. Maybe we can even set flake8 to allow max-line length as 120 for example, like we have in Patroni.